addons/arsenal/functions/fnc_fillLoadoutsList.sqf

```diff
@@ -1,42 +1,60 @@
 #include "..\script_component.hpp"
 #include "..\defines.hpp"
 /*
- * Author: Alganthe, johnb43
+ * Author: Alganthe, johnb43, LinkIsGrim
  * Fill loadouts list.
```

Comment:
Maybe add something in the description about the function filling 10 loadouts per frame?